docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_Fault_Tolerance_SmallRye.adoc

```diff
@@ -44,43 +44,29 @@ This MicroProfile specification introduces the following interceptor bindings:
 * `@Bulkhead`
 * `@Asynchronous`
 
-For usage please refer to MicroProfile Fault Tolerance 2.0 https://download.eclipse.org/microprofile/microprofile-fault-tolerance-2.0/microprofile-fault-tolerance-spec.html[specification].
+For usage please refer to MicroProfile Fault Tolerance 2.1 https://download.eclipse.org/microprofile/microprofile-fault-tolerance-2.1/microprofile-fault-tolerance-spec.html[specification].
 
-== Hystrix Configuration
+== Configuration
 
-MicroProfile Fault Tolerance extension is implemented by the SmallRye Fault Tolerance project which uses https://github.com/Netflix/Hystrix[Hystrix] as an underlying library.
+Apart from configuration properties defined by the specification, the SmallRye implementation provides the following
+configuration properties:
 
-MicroProfile Config is used to configure Hystrix properties.
-The integration module picks up all `hystrix` prefixed properties to configure the library.
-Please refer to https://github.com/Netflix/Hystrix/wiki/Configuration[configuration for configurable properties].
+.SmallRye Fault Tolerance configuration properties
+|===
+|Name |Default |Description
 
-NOTE: Deployments can supply configuration for Hystrix using the defined `META-INF/microprofile-config.properties` file.
-      However, when multiple MicroProfile Fault Tolerance deployments are deployed in a single WildFly instance and since configuration of Hystrix is singleton
-      it is recommended to use globally scoped properties in this case. Please refer to MicroProfile Config subsystem how to configure `ConfigSource`-s.
+|`io.smallrye.faulttolerance.globalThreadPoolSize`
+|100
+|Number of threads used by the the fault tolerance mechanisms. This does not include bulkhead thread pools.
```

Comment:
the the -> the

Reply:
Will by fixed by the next push - the wording was wrong upstream - will be fixed there too now https://github.com/smallrye/smallrye-fault-tolerance/pull/200